class.jetpack.php

```diff
@@ -5228,6 +5228,9 @@ public static function clean_nonces( $all = false ) {
 	 * @param bool $restate private
 	 */
 	public static function state( $key = null, $value = null, $restate = false ) {
+		if ( XMLRPC_REQUEST ) {
```

Comment:
Should be either `defined( 'XMLRPC_REQUEST' )` or `defined('XMLRPC_REQUEST') && XMLRPC_REQUEST )`